packages/lazy/src/index.ts

```diff
@@ -14,6 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+const valueKey = Symbol("value")
+const initKey = Symbol("init")
```

Comment:
Creating new symbols for each call wastes memory for no reason so they're moved out of the function scope. This wasn't an issue with the limited number of calls but better fix it before it potentially becomes one.